ui/apps/platform/cypress/integration/vulnerabilities/VulnerabilityReporting/vulnerabilityReportingNav.test.ts

```diff
@@ -0,0 +1,104 @@
+        cy.url().should('include', reportingBasePath);
+        cy.get('h1').should('contain', 'Vulnerability reporting');
+    });
+
+    it('navigates to the create page', () => {
+        cy.contains('button', 'Create report').click();
+        cy.url().should('include', `${reportingBasePath}?action=create`);
+        cy.get('h1').should('contain', 'Create report');
+    });
+
+    it('navigates to the details page', () => {
+        cy.get('@reports')
+            .its('response')
+            .then((response) => {
+                const firstReport = response.body.reportConfigs[0];
+                cy.intercept('GET', `/v2/reports/configurations/${firstReport.id}`, firstReport).as(
+                    'reportDetails'
+                );
+
+                cy.contains('a', firstReport.name).click();
+                cy.wait('@reportDetails');
+
+                cy.url().should('include', `${reportingBasePath}/${firstReport.id}`);
+                cy.get('h1').should('contain', firstReport.name);
+            });
+    });
+
+    it('navigates to the edit page', () => {
+        cy.get('@reports')
+            .its('response')
+            .then((response) => {
+                const firstReport = response.body.reportConfigs[0];
+                cy.intercept('GET', `/v2/reports/configurations/${firstReport.id}`, firstReport).as(
+                    'reportDetails'
+                );
+
+                cy.get('table .pf-v5-c-menu-toggle').first().click();
+                cy.contains('button', 'Edit report').click();
+
+                cy.wait('@reportDetails');
+                cy.url().should('include', `${reportingBasePath}/${firstReport.id}?action=edit`);
+                cy.get('h1').should('contain', 'Edit report');
+
+                cy.get('.pf-v5-c-breadcrumb__link').click();
+                cy.contains('a', firstReport.name).click();
+
+                cy.contains('button', 'Actions').click();
+                cy.contains('button', 'Edit report').click();
+
+                cy.url().should('include', `${reportingBasePath}/${firstReport.id}?action=edit`);
+                cy.get('h1').should('contain', 'Edit report');
+            });
+    });
+
+    it('navigates to the clone page', () => {
+        cy.get('@reports')
+            .its('response')
+            .then((response) => {
+                const firstReport = response.body.reportConfigs[0];
+                cy.intercept('GET', `/v2/reports/configurations/${firstReport.id}`, firstReport).as(
+                    'reportDetails'
+                );
+
+                cy.get('table .pf-v5-c-menu-toggle').first().click();
+                cy.contains('button', 'Clone report').click();
+
+                cy.wait('@reportDetails');
+                cy.url().should('include', `${reportingBasePath}/${firstReport.id}?action=clone`);
+                cy.get('h1').should('contain', 'Clone report');
+
+                cy.get('.pf-v5-c-breadcrumb__link').click();
```

Comment:
A couple of suggestions for more semantic selectors:
```ts
                cy.get('nav a').click();
                cy.get('nav[aria-label="Breadcrumb"] a').click();
                cy.get('nav a:contains("Vulnerability reporting")').click();
                cy.get('a:contains("Vulnerability reporting")').click();
```
There are a lot of options I think would work in this case, but I would probably be partial to the second one. It communicates what action a user would be taking a bit better, and is less dependent on PatternFly internals. e.g. This selector should still work after upgrading PatternFly.

Reply:
I'll admit I knew having `pf-v5` as a selector was a red flag, but I went with the easy route for the first draft. I still don't think I would've came up with all of these options though, I definitely prefer 2 & 3.